include/roboptim/core/util.hxx

```diff
@@ -170,7 +170,7 @@ namespace roboptim
     ROBOPTIM_ASSERT (startRow + b.rows () <= m.rows ());
     ROBOPTIM_ASSERT (startCol + b.cols () <= m.cols ());
 # if __cplusplus < 201103L || !defined (__GXX_EXPERIMENTAL_CXX0X__)
-    ROBOPTIM_ASSERT (int (M::IsRowMajor) == int (B::IsRowMajor));
+    ROBOPTIM_ASSERT ( ROBOPTIM_UNUSED int (M::IsRowMajor) == int (B::IsRowMajor));
```

Comment:
So actually, the ROBOPTIM_UNUSED was moved by François when the assertion was still made by BOOST_STATIC_ASSERT. You removed the ROBOPTIM_UNUSED when you switched to ROBOPTIM_ASSERT, so I guess it's a mistake I did when merging.
Can you confirm the ROBOPTIM_UNUSED should be completely removed?